### PR TITLE
Prepare for linear rao loopflow regression test

### PR DESCRIPTION
**Pull request by pengbo-ruidian**
**Please check if the PR fulfills these requirements** *(please use `'[x]'` to check the checkboxes, or submit the PR and then click the checkboxes)*
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem ?** *If so, link to this issue using `'#XXX'` and skip the rest*

**What kind of change does this PR introduce?** *(Bug fix, feature, docs update, ...)*

**What is the current behavior?** *(You can also link to an open issue here)*

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change?** *(What changes might users need to make in their application due to this PR?)*

**Other information**:

(if any of the questions/checkboxes don't apply, please delete them entirely)

### CODE REVIEW

File: ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java [hunk 1]
```diff
@@ -103,6 +103,11 @@ public String getVersion() {
             linearOptimisationEngine.run(linearRaoData, linearRaoParameters);
             LOGGER.info("Iteration {} - linear optimization [end]", iteration);
 
+            if (!linearOptimisationEngine.getSolverResultStatusString().equals("OPTIMAL")) {
+                LOGGER.info("Iteration {} - linear optimization is infeasible", iteration); //handle INFEASIBLE solver status
+                break;
+            }
+
```
Comment (alexandrem-rte) [inline ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java:110]: Why did you add this check? Is it a behaviour specific to loopflow computation?

File: data/glsk/glsk-file/src/main/java/com/farao_community/farao/data/glsk/import_/UcteGlskDocument.java [hunk 2]
```diff
@@ -41,6 +41,10 @@
      * List of Glsk point by country code
      */
     private Map<String, List<GlskPoint>> ucteGlskPointsByCountry; //map <CountryID, List<GlskPoint>>
+    /**
+     * document GSKTimeInterval
+     */
+    private Interval gSKTimeInterval; // GSKTimeInterval. ex: <GSKTimeInterval v="2016-07-28T22:00Z/2016-07-29T22:00Z"/>
```
Comment (alexandrem-rte) [inline data/glsk/glsk-file/src/main/java/com/farao_community/farao/data/glsk/import_/UcteGlskDocument.java:47]: Typo here I think it should be GskTimeInterval

File: ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java (hunk 1 above)
Comment (pengbo-ruidian) [reply ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRao.java:110]: Non, it is not specific to loopflow computation.

**Comment by pengbo-ruidian** [comment]
> 
> 
> I have 1 comment and 1 question. Besides I think we should not add josiris-server.log in the source code

removed
